public/home.php

```diff
@@ -1,5 +1,5 @@
 <?php
-// SPDX-License-Identifier: GPL-2.0
+ SPDX-License-Identifier: GPL-2.0
```

Comment:
This is not a fix, you just broke the home.php. It's literally a comment, please, test first before submit. I don't want to break production site just because of this stupid mistake.

Why did you even uncomment this line?

Reply:
Sorry, my bad. I have fix it.